app/src/main/java/org/p2p/wallet/send/SendFeeRelayerManager.kt

```diff
@@ -325,15 +325,13 @@ class SendFeeRelayerManager(
                 return FeeCalculationState.NoFees
             }
 
-            val feePayerTokensMints = sendServiceRepository.getCompensationTokens()
-
             val poolsStateFees = getFeesInPayingTokenUseCase.execute(
                 feePayerToken = feePayerToken,
                 transactionFeeInSol = fees.transactionFee,
                 accountCreationFeeInSol = fees.accountCreationFee
             )
 
-            if (poolsStateFees != null && feePayerToken.mintAddressB58 in feePayerTokensMints) {
```

Comment:
we don't need this check here because finding alternative tokens already checks for supported mints